www/html/ci/application/controllers/Landing.php

```diff
@@ -3,15 +3,28 @@
 
 class Landing extends CI_Controller {
 
+	public $loader;
+	public $twig;
+
 	public function __construct() {
 		parent::__construct();
 
 		// load database
         $this->load->database();
 		$this->load->helper('url');
+
+		//setup twig 
+		$this->loader = new Twig_Loader_Filesystem('ci/application/views');
+		$this->twig = new Twig_Environment($this->loader);
 	}
 	
 	public function index() {
+		
+		require_once '/../../../../../vendor/autoload.php';
```

Comment:
anything outside of www/html is not part of the actual site. Can this be moved inside www/html somehow.